src/execute_request.jl

```diff
@@ -29,6 +30,9 @@ function display_dict(x)
     if mimewritable(application_vnd_vegalite_v2, x)
         data[string(application_vnd_vegalite_v2)] = JSON.parse(limitstringmime(application_vnd_vegalite_v2, x))
     end
+    if mimewritable(application_vnd_dataresource, x)
+        data[string(application_vnd_dataresource)] = JSON.parse(limitstringmime(application_vnd_dataresource, x))
```

Comment:
Please use `JSONText(limitstringmime(application_vnd_dataresource, x))` (using https://github.com/JuliaIO/JSON.jl/pull/237), and similarly for application_vnd_vegalite_v2.

Also update the REQUIRE file to list `JSON 0.17.0`

Reply:
Done.